docs/content/guides/how-to-write-guidance/how-to-present-guidance.mdx

```diff
@@ -139,9 +139,9 @@ Where guidance is relevant to all users, you can provide concise guidance above
 	<Text>All fields are required unless marked optional.</Text>
 
 	<ControlGroup label="Label 1" hint="Hint text" block>
-		<Radio checked={false}>Phone</Radio>
-		<Radio checked={false}>Tablet</Radio>
-		<Radio checked={true}>Laptop</Radio>
```

Comment:
It would be nice to have this as `defaultChecked={true}`. To allow consumers to do the same we'd need to add it as a prop or TS will complain. What do you think?

Reply:
The example itself doesn't really benefit from having a checked option. You could argue that it's actually a negative form of guidance to have an optional group of radio buttons and have 1 selected by default which could get missed?

If we add the prop, would we want teams using it? It feels like most teams would use it in a controlled way, so `checked` is fine?

Reply:
Spoke offline and have created a ticket to add `default...` to all components that support uncontrolled state.